src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -52,6 +52,8 @@
 
     private String resourceRequestCpu;
 
+    private Long runAsUser;
```

Comment:
Not `Integer`?

Reply:
in fabric8 it's long so I'm keeping it long